lago_images/images.py

```diff
@@ -119,6 +119,36 @@ def custom_build_action(self, *args, **kwargs):
             commands_file=self.spec.commands_file
         )
 
+        if self.spec.size != 0:
+            build_utils.resize_disk_image(
+                dst_image=self.dst_path,
+                new_size=self.spec.size,
+            )
+
+            build_utils.copy_image_with_cp(
+                src_image=self.dst_path,
+                dst_image=self.dst_path+"_new",
+            )
+
+            build_utils.resize_image_partition(
+                dst_image=self.dst_path,
+                new_dst_image=self.dst_path+"_new",
+                partition=self.spec.expand
+            )
+
+            build_utils.remove_disk_image(
```

Comment:
This should be called inside of a `finally`, just in case the previous commands were failed.